capstone-app/src/app/modules/photos/components/photos-section/photos-section.component.html

```diff
@@ -28,8 +28,7 @@
         [formControl]="limitControl"
         min="1"
         class="limit-input"
-        max="{{ maxPhotos }}"
-      />
+        max="{{ maxPhotos }}"/>
```

Comment:
Nit: The trailing slash on things like input and img are from a previous era and are no longer necessary. Current style is to not use them.